0002-fix-installation-with-flatpak.patch

```diff
@@ -0,0 +1,37 @@
+From 756e4ecf3737660019ae1282a5a70c688e5ad274 Mon Sep 17 00:00:00 2001
+From: Martin Marmsoler <martin.marmsoler@gmail.com>
+Date: Wed, 2 Nov 2022 17:27:57 +0100
+Subject: [PATCH 2/2] fix installation with flatpak
+
+---
+ source6/ChartGeany/ChartGeany.pro | 13 +++++--------
+ 1 file changed, 5 insertions(+), 8 deletions(-)
+
+diff --git a/source6/ChartGeany/ChartGeany.pro b/source6/ChartGeany/ChartGeany.pro
+index 39aad23..0fc65c5 100644
+--- a/source6/ChartGeany/ChartGeany.pro
++++ b/source6/ChartGeany/ChartGeany.pro
+@@ -129,15 +129,12 @@ QMAKE_CXXFLAGS += -fomit-frame-pointer -fstack-protector
+ # QMAKE_STRIP = echo
+ QMAKE_POST_LINK += strip $(TARGET)
+ 
+-target.path = /opt/$$PKG_NAME
+-images.path = /opt/$$PKG_NAME
+-desktop.path = /usr/share/applications/
+-images.files = images/$${PKG_NAME}.png
++target.path = $${PREFIX}/bin
++desktop.path = $${PREFIX}/share/applications/
+ desktop.files = $${PKG_NAME}.desktop
+-chartgeany.path = /opt/$$PKG_NAME
+-chartgeany.commands = rm -rf /usr/bin/$$TARGET; ln -s /opt/$$PKG_NAME/$$TARGET /usr/bin
+-chartgeany.uninstall = rm -rf /usr/bin/$$TARGET
+-INSTALLS += target desktop images chartgeany
++metainfo.path = $${PREFIX}/share/metainfo/
++metainfo.files = $${PGK_NAME}.metainfo.xml
```

Comment:
```suggestion
+metainfo.files = $${PKG_NAME}.metainfo.xml
```

Reply:
again here it should be installed with the right name from the get go. There is strictly no reason to do otherwise.

Reply:
ok I will change